src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -219,7 +221,11 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
       if chatDetails.communityId != "":
         communityChats = self.controller.getCommunityById(chatDetails.communityId).chats
 
-      let renderedMessageText = self.controller.getRenderedText(message.parsedText, communityChats)
+      var renderedMessageText = self.controller.getRenderedText(message.parsedText, communityChats)
+
+      if (prevMessage.contentType.ContentType == ContentType.Image and prevMessage.messageText == renderedMessageText):
+        renderedMessageText = ""
```

Comment:
Can you explain why you do that?

Reply:
if someone send several images with text from mobile app, we receive it as several image messages with same text. I skip text from images for removing duplications and display this text just for last one.

Reply:
Ah ok, got it. Could you add that as a comment here, that way we know what it does. Thanks